### ISSUE TITLE
BetterVolume Right clicking users crashes discord when plugin is enabled

### ISSUE THREAD

**Issue by NeverEndingCycle** [Zerthox/BetterDiscord-Plugins#147]
### Bug description

Right clicking users crashes discord when plugin is enabled

### Plugin

Better Volume

### Plugin version

2.4.0

### Discord client

Stable

**Comment by zacharee** [comment]
Logs:

```
[2023-04-05 00:13:38.915][CONSOLE:WARN] [BetterDiscord] [WebpackModules~getModule] Module filter threw an exception. t=>{if(null==t||!t.toString||"function"!=typeof(null==t?void 0:t.toString))return;let n="";try{n=null==t?void 0:t.toString([])}catch(e){n=null==t?void 0:t.toString()}if(!n)return!1;for(const t of e)if(!n.includes(t))return!1;return!0} Illegal invocation
TypeError: Illegal invocation
    at betterdiscord/renderer.js:4:44927
    at betterdiscord/renderer.js:4:45697
    at s.getModule (betterdiscord/renderer.js:4:46729)
    at getModule (betterdiscord/renderer.js:4:366446)
    at eval (betterdiscord://plugins/VoiceActivity.plugin.js:371:24)
    at buildPlugin (betterdiscord://plugins/VoiceActivity.plugin.js:1046:4)
    at eval (betterdiscord://plugins/VoiceActivity.plugin.js:1058:46)
    at A.requireAddon (betterdiscord/renderer.js:4:35265)
    at A.loadAddon (betterdiscord/renderer.js:4:7766)
    at A.loadAddon (betterdiscord/renderer.js:4:32937)
    at A.loadAllAddons (betterdiscord/renderer.js:4:10240)
    at A.initialize (betterdiscord/renderer.js:4:4662)
    at A.initialize (betterdiscord/renderer.js:4:32136)
    at Object.startup (betterdiscord/renderer.js:4:357163)
[2023-04-05 00:13:38.934][CONSOLE:WARN] [BetterDiscord] [WebpackModules~getModule] Module filter threw an exception. t=>{if(null==t||!t.toString||"function"!=typeof(null==t?void 0:t.toString))return;let n="";try{n=null==t?void 0:t.toString([])}catch(e){n=null==t?void 0:t.toString()}if(!n)return!1;for(const t of e)if(!n.includes(t))return!1;return!0} Illegal invocation
TypeError: Illegal invocation
    at betterdiscord/renderer.js:4:44927
    at betterdiscord/renderer.js:4:45697
    at s.getModule (betterdiscord/renderer.js:4:46729)
    at getModule (betterdiscord/renderer.js:4:366446)
    at eval (betterdiscord://plugins/VoiceActivity.plugin.js:432:19)
    at buildPlugin (betterdiscord://plugins/VoiceActivity.plugin.js:1046:4)
    at eval (betterdiscord://plugins/VoiceActivity.plugin.js:1058:46)
    at A.requireAddon (betterdiscord/renderer.js:4:35265)
    at A.loadAddon (betterdiscord/renderer.js:4:7766)
    at A.loadAddon (betterdiscord/renderer.js:4:32937)
    at A.loadAllAddons (betterdiscord/renderer.js:4:10240)
    at A.initialize (betterdiscord/renderer.js:4:4662)
    at A.initialize (betterdiscord/renderer.js:4:32136)
    at Object.startup (betterdiscord/renderer.js:4:357163)
[2023-04-05 00:13:38.971][CONSOLE:WARN] [BetterDiscord] [WebpackModules~getModule] Module filter threw an exception. (m) => m.toString?.().includes("().dividerDefault") Illegal invocation
TypeError: Illegal invocation
    at getModule$1.searchExports (betterdiscord://plugins/VoiceActivity.plugin.js:739:53)
    at betterdiscord/renderer.js:4:45697
    at s.getModule (betterdiscord/renderer.js:4:46729)
    at getModule (betterdiscord/renderer.js:4:366446)
    at eval (betterdiscord://plugins/VoiceActivity.plugin.js:739:22)
    at buildPlugin (betterdiscord://plugins/VoiceActivity.plugin.js:1046:4)
    at eval (betterdiscord://plugins/VoiceActivity.plugin.js:1058:46)
    at A.requireAddon (betterdiscord/renderer.js:4:35265)
    at A.loadAddon (betterdiscord/renderer.js:4:7766)
    at A.loadAddon (betterdiscord/renderer.js:4:32937)
    at A.loadAllAddons (betterdiscord/renderer.js:4:10240)
    at A.initialize (betterdiscord/renderer.js:4:4662)
    at A.initialize (betterdiscord/renderer.js:4:32136)
    at Object.startup (betterdiscord/renderer.js:4:357163)
```

**Comment by AceLikesGhosts** [comment]
Same issue here, although a different stacktrace:
```
184b1a0f1f64c5040ae4.js:241 Error: Menu API only allows Items and groups of Items as children. Received ()=>null (object) instead
    at 0192aa8c9176cfd4091d.js:2948
    at Array.reduce (<anonymous>)
    at Ki (0192aa8c9176cfd4091d.js:2947)
    at 0192aa8c9176cfd4091d.js:2947
    at Array.reduce (<anonymous>)
    at Ki (0192aa8c9176cfd4091d.js:2947)
    at Xi (0192aa8c9176cfd4091d.js:2950)
    at betterdiscord/renderer.js:4
    at Sa (0192aa8c9176cfd4091d.js:1744)
    at gu (0192aa8c9176cfd4091d.js:1798)
    at Ic (0192aa8c9176cfd4091d.js:1791)
    at mc (0192aa8c9176cfd4091d.js:1791)
    at hc (0192aa8c9176cfd4091d.js:1790)
    at oc (0192aa8c9176cfd4091d.js:1787)
    at uc (0192aa8c9176cfd4091d.js:1788)
    at Fi (0192aa8c9176cfd4091d.js:1727)
    at 0192aa8c9176cfd4091d.js:1785
```

**Comment by callmeyonggor** [comment]
Issue solved in Betterdiscord v1.9.2
